docs/userguide/src/en/bpmn/ch02-GettingStarted.adoc

```diff
@@ -525,7 +525,7 @@ If you get back a proper json response, the REST API is up and running.
 The first step is to deploy a process definition. With the REST API, this is done by uploading a .bpmn20.xml file (or .zip file for multiple process definitions) as 'multipart/formdata':
 
 ----
-curl --user rest-admin:test -F "file=@holiday-request.bpmn20.xml" http://localhost:8080/flowable-rest/service/repository/deployments
+curl --user rest-admin:test -X POST -F "file=@holiday-request.bpmn20.xml" http://localhost:8080/flowable-rest/service/repository/deployments
```

Comment:
I believe that the correct location for this change is `docs/docusaurus/docs/bpmn/ch02-GettingStarted.md` as the current online documentation is using `docusaurus` and not the `adoc` versions.

Reply:
Very glad to hear from you, I have moved the modification to `docs/docusaurus/docs/bpmn/ch02-GettingStarted.md`